testsuite/features/secondary/min_salt_minions_page.feature

```diff
@@ -54,7 +54,7 @@ Feature: Management of minion keys
 
   Scenario: The minion communicates with the Salt master
     Given I am authorized as "testing" with password "testing"
```

Comment:
Off-topic: Some places we use `Given I am authorized` as a short path for  `Given I am authorized as "testing" with password "testing"`. It might be good to always use the same, my suggestion is to use the long step, at least for me is more clear.

Reply:
OK. Then we should also remove the step definition of `I am authorized` to prevent further usage.

Would you prepare a PR? :trollface:

Reply:
Yep, if we agree I will. Maybe on Christmas xD

Reply:
as a gift ? :gift: :yum: